app/store/configureStore.dev.ts

```diff
@@ -1,13 +1,11 @@
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
 /* eslint-disable @typescript-eslint/no-var-requires */
-import { createStore, applyMiddleware, compose } from 'redux';
+import { createStore, applyMiddleware, compose } from '@reduxjs/toolkit';
```

Comment:
Nice! How to you like Redux toolkit?

Reply:
Mostly like it for the typed action creators, but there are some other cool features.